src/main/java/org/autopotato/linkedinclonebackend/LinkedinCloneBackendApplication.java

```diff
@@ -2,11 +2,33 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
 
 @SpringBootApplication
 public class LinkedinCloneBackendApplication {
 
     public static void main(String[] args) {
         SpringApplication.run(LinkedinCloneBackendApplication.class, args);
     }
+
+    /**
+     * Configure CORS settings to all mappings, origins and headers. Allow only GET and POST requests
+     * @return WebMvcConfigurer with the necessary settings
+     */
+    @Bean
+    public WebMvcConfigurer corsConfigurer() {
+        return new WebMvcConfigurer() {
+
+            @Override
+            public void addCorsMappings(CorsRegistry registry) {
+                registry
+                    .addMapping("/**")
+                    .allowedOrigins("*")
```

Comment:
Let's limit the origins to only `http://localhost:4200`.

Reply:
Ok I’ll change it as soon as I get home